driver-core/src/test/functional/com/mongodb/client/model/expressions/ComparisonExpressionsFunctionalTest.java

```diff
@@ -60,6 +60,17 @@ class ComparisonExpressionsFunctionalTest extends AbstractExpressionsFunctionalT
             of(Instant.now())
     );
 
+    @Test
+    public void literalsTest() {
```

Comment:
Switch handles nulls, so I had a closer look at ofNull. I saw that there was no null literal check, and I added it here.

I generally add a missing tests for something I was using in a PR in that same PR, rather than a new PR. Let me know if there's a better way to handle this.